praca.tex

```diff
@@ -139,7 +139,7 @@
   showstringspaces=false,
   numberstyle=\tiny,
   frame=l,
-  escapeinside={\%*}{*)},            % if you want to add LaTeX within your code
+  escapeinside={\%*}{*},            % if you want to add LaTeX within your code
```

Comment:
Jak już to zmieniasz to może na coś bardziej sensownego, np. (* i *)?

Reply:
Google pokazuje, że popularne w urzyciu jest `escapeinside={(*@}{@*)}`

Reply:
Trzy znaki to lekka przesada, zdaje mi się, `*@` i `@*` podobają mi się najbardziej.